pychromecast/__init__.py

```diff
@@ -20,7 +20,7 @@
     start_discovery,
     stop_discovery,
 )
-from .dial import get_device_status, reboot, DeviceStatus
+from .dial import get_device_status, DeviceStatus
```

Comment:
Does `get_device_status` still work ?

Reply:
Yes, it still works, but it's crippled and doesn't reveal anything not revealed through mDNS discovery.
It could be removed, but then mDNS would be mandatory.